marv_robotics/nodes/camera_frames.py

```diff
@@ -59,7 +60,7 @@ def camera_frames(bagmeta, messages, image_width=320, max_frames=50):
             continue
 
         try:
-            img = imgmsg_to_cv2(msg, "rgb8")
+            img = imgmsg_to_cv2(msg, encoding)
```

Comment:
If I'm not mistaken, then the argument you specify here is the "target" encoding, why do you want to change that?
The encoding that the ROS image has is already part of the Image msg and should be already considered by the conversion function....

Reply:
All video images in my Marv instance show a R/B and B/R confusion as well. Using bgr8 as target encoding makes sense to me. OpenCV uses BGR encoding internally and methods like cv2.imwrite want BGR encoded images.

Reply:
@Earthwings, do you use the latest release. If so, I would check which code path is taken here https://github.com/ternaris/marv-robotics/blob/master/marv_robotics/cam.py#L124-L130 and open a new issue in case the bags look fine e.g. in image_view or rviz.

Reply:
@NikolausDemmel Works indeed fine in 17.11.
Unfortunately I'm running into multiple other bugs with that release, but that's off topic here.